iterative/utils/analytics.go

```diff
@@ -349,7 +349,7 @@ func JitsuEventPayload(action string, e error, extra map[string]interface{}) (ma
 	}
 
 	if e != nil { 
-		// Get the type of the error as a string, but not the error message itself, because it can be sensitive.
+		// Only use error type (not message) to avoid potentially sensitive information.
```

Comment:
lol

```suggestion
		// Only use error type (not message) to avoid potentially sensitive information
```

Reply:
[I do.](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences)

Reply:
Feel free to revert 🙈

Reply:
> _Feel free to revert_

Thanks! It's not worth the hassle to my mind, but appreciate it anyway. 🙏🏼 

> _I'm not siding with you, and I don't like it, but I am happily caving in to the concept of an official spec. Do you like my full sentences?_

This “rule” is quite flexible and not anybody follows it, less yet for inline code comments; it's mostly meant to document declarations. It's just one of those acquired tastes that some Gophers develop over time. 🦦